build-system/common/exec.js

```diff
@@ -31,7 +31,7 @@ const shellCmd = process.platform == 'win32' ? 'cmd' : '/bin/bash';
  * object.
  *
  * @param {string} cmd
- * @param {?Object} options
+ * @param {?Object=} options
```

Comment:
Curious: If this is being changed, would it make sense to change the signature as follows?

```js
function exec(cmd, options = {'stdio': 'inherit'}) {
```

(Same question applies to other instances)

Reply:
I think that would make sense.